.eslintrc.yml

```diff
@@ -23,7 +23,7 @@ rules:
   no-empty-function: "off"
   no-use-before-define: ["error", { "functions": false }]
   # disable everything, except Rest/Spread Properties in ES2018
-  es/no-async-iteration: "error"
+  es/no-async-iteration: "error"d
```

Comment:
intended?

Reply:
no, only edited the doc file but I guess I fat fingered a letter in there before I switched to the doc file, only noticed after I pushed it